examples/matplotlib/mpl_plot_distcomp_combinedims.py

```diff
@@ -0,0 +1,5 @@
+import matplotlib.pyplot as plt
+import arviz as az
+
+data = az.load_arviz_data('rugby')
+az.plot_dist_comparison(data, var_names=["defs"], coords={"team" : ["Italy"]}, combine_dims=[''])
```

Comment:
Here combine_dims is not doing anything. As we are subsetting data to use only the italy coordinate value, there are no more dims that could be combined.